browser/ui/color/brave_color_id.h

```diff
@@ -120,7 +120,8 @@
     E_CPONLY(kColorBraveVerticalTabNTBShortcutTextColor)
 
 #define BRAVE_SPLIT_VIEW_COLOR_IDS \
-    E_CPONLY(kColorBraveSplitViewTileBackground)
+    E_CPONLY(kColorBraveSplitViewTileBackground)        \
```

Comment:
Just wondering if this should use the Nala focus color?

Reply:
I think tile background should be kept as is, because it can be affected easily by custom theme.

Reply:
Ah, if what you meant to say is the static color usage for the focus color, it was what's in the design guide.